router/src/server.rs

```diff
@@ -208,40 +207,30 @@ async fn get_chat_tokenize(
 
     let input = generate_request.inputs.clone();
     let encoding = infer.tokenize(generate_request).await?;
-    if let Some(encoding) = encoding {
-        let tokens: Vec<SimpleToken> = encoding
-            .get_ids()
-            .iter()
-            .zip(encoding.get_offsets())
-            .map(|(&id, &(start, stop))| {
-                let text = input
-                    .chars()
-                    .skip(start)
-                    .take(stop - start)
-                    .collect::<String>();
-                SimpleToken {
-                    id,
-                    text,
-                    start,
-                    stop,
-                }
-            })
-            .collect();
+    let tokens: Vec<SimpleToken> = encoding
+        .get_ids()
+        .iter()
+        .zip(encoding.get_offsets())
```

Comment:
This zip will fail for Python.

Reply:
I "fixed" it by ignoring offsets if not present.

Python non rust will definitely exist in forms where offsets are not available (ByT5) so I'll assume we cannot have them.
In that case we'll send the raw ids and totally ignore the "text" part (since it doesn't have a good definition without offsets in the original query)